src/config.rs

```diff
@@ -695,16 +697,41 @@ impl Default for MetricConfig {
     }
 }
 
+// This type exists purely for interacting with Serde, and using `slog::Level` should
+// be preferred.
 #[derive(Serialize, Deserialize)]
-#[serde(remote = "LogLevelFilter")]
+#[serde(remote = "slog::Level")]
 #[serde(rename_all = "kebab-case")]
-pub enum LogLevel {
+pub enum Level {
```

Comment:
Why not use `FilterLevel`? What's the difference?

Reply:
It's a `Drain` which  is a target for logging, it is produced when you call [`Drain::filter_level`](https://docs.rs/slog/2.2.3/slog/trait.Drain.html#method.filter_level).

I'm not sure I understand what you are suggesting I do. :frowning:

Reply:
That's not what [I referred](https://docs.rs/slog/2.2.3/slog/enum.FilterLevel.html).

Reply:
[`Drain::filter_level`](https://docs.rs/slog/2.2.3/slog/trait.Drain.html#method.filter_level) accepts a `Level` which we call [here](https://github.com/pingcap/tikv/pull/3010/files#diff-6e29d25af398da79e2713e1061c7706aR40).

I'm not entirely clear what the difference is, but [here is their definition](https://docs.rs/slog/2.2.3/src/slog/lib.rs.html#2038-2078) and it looks like `Level` is preferred.

Reply:
seem that we need to define a new Drain or pass a filter like https://docs.rs/slog/2.2.3/slog/trait.Drain.html#method.filter if we want to use FilterLevel directly.

Btw, the FilterLevel also uses `warning`, we still need to do the `warn` -> `warning` conversation for the compatibility. If there is no extra benefit and we don't care `off`, I prefer `Level` here.